src/openseadragon.js

```diff
@@ -159,7 +159,7 @@
   * @property {Boolean} [debugMode=false]
   *     TODO: provide an in-screen panel providing event detail feedback.
   *
-  * @property {String} [debugGridColor='#437AB2']
+  * @property {String} [debugGridColor=['#437AB2']]
```

Comment:
Should provide documentation here for the different options. 

Also, we might as well provide a set of colors for default... maybe 10 or 12? Most of the time people won't care exactly which colors, they'll just want them to be different, so we might as well make it easy for them.